app/assets/stylesheets/sprangular/_base.scss

```diff
@@ -42,6 +42,15 @@ body {
   }
 }
 
+.products-loading {
+  background: transparent url(loading.gif) center no-repeat;
```

Comment:
Color literals like `transparent` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>URLs should be enclosed in quotes